src/com/tt/TrayUI.java

```diff
@@ -85,15 +85,15 @@ public void mouseClicked(MouseEvent e) {
                             malformedURLException.printStackTrace();
                         }
 
-                        JLabel centerLabel = new JLabel("<html><center><span style='color:white;font-size:55px;'>" + weatherAPIKeyValue[0] + "°C" + "</span></center></html>", i, SwingConstants.LEFT);
+                        JLabel centerLabel = new JLabel("<html><center><span style='color:white;font-size:55px;'>" + Integer.parseInt(weatherAPIKeyValue[0]) + "°C" + "</span></center></html>", i, SwingConstants.LEFT);
```

Comment:
The numbers that we get from the API are floating point numbers so if you try parsing these number as Integer you'll get NumberFormatException. I would suggest you to parse these numbers as Float using Float.parseFloat() method and enclose this statement inside Math.ceil() function which will give you an upperbound Integer.